enroll/service.go

```diff
@@ -145,3 +148,57 @@ func (svc service) Enroll(ctx context.Context) (Profile, error) {
 
 	return *profile, nil
 }
+
+func (svc service) OTAEnroll(ctx context.Context) (Payload, error) {
+	payload := NewPayload("Profile Service")
+	payload.PayloadIdentifier = "com.github.micromdm.ota.profile-service"
+	payload.PayloadDisplayName = "MicroMDM Profile Service"
+	payload.PayloadDescription = "Profile Service enrollment"
+	payload.PayloadOrganization = "MicroMDM"
+	payload.PayloadContent = ProfileServicePayload{
+		URL:              svc.URL + "/ota/phase23",
+		Challenge:        "",
+		DeviceAttributes: DefaultProfileServiceDeviceAttributes,
+	}
+
+	// yes, this is a bare Payload, not a Profile
+	return *payload, nil
+}
+
+func (svc service) OTAPhase2(ctx context.Context) (Profile, error) {
+	profile := NewProfile()
+	profile.PayloadIdentifier = "com.github.micromdm.micromdm.ota-scep"
+	profile.PayloadOrganization = "MicroMDM"
+	profile.PayloadDisplayName = "Enrollment Profile"
+	profile.PayloadDescription = "The server may alter your settings"
+	profile.PayloadScope = "System"
+
+	scepContent := SCEPPayloadContent{
+		URL:      svc.SCEPURL,
+		Keysize:  1024,
+		KeyType:  "RSA",
+		KeyUsage: 0,
```

Comment:
Also: again merely copied it from the enrollment profile above. But: the bit fields appear to match, can use constants.

Note the SCEP profile docs from Apple say:

> Some certificate authorities, such as Windows CA, support only encryption or signing, but not both at the same time.